tests/CMakeLists.txt

```diff
@@ -223,6 +223,7 @@ set(QUDA_CTEST_LAUNCH ${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} ${MPIEXEC_MA
 if(QUDA_DIRAC_WILSON
    OR QUDA_DIRAC_CLOVER
```

Comment:
+ `QUDA_DIRAC_CLOVER_HASENBUSCH`? Or does that already imply `QUDA_DIRAC_CLOVER`?

Reply:
Indeed, it already implies both regular clover and twisted clover.